huggingface/pytorch/tgi/docker/0.6.0/py3/cu118/THIRD-PARTY-LICENSES

```diff
@@ -0,0 +1,495 @@
+** text-generation-inference; version 0.6.0 -- https://github.com/huggingface/text-generation-inference
```

Comment:
could we use curl to copy the file instead of check in here?

Reply:
Changed.